File: TOI-Zero_68/A1/018/018.c
#include <stdio.h>

int main() {
    int a;
    scanf("%d", &a);
    
    if (a > 9 || a == 0) {
        printf("Error : Out of range\n");
    } else if (a < 0) {
        printf("Error : Please input positive number\n");
    } else {
        const char* roman_numerals[] = {"I", "II", "III", "IV", "V", "VI", "VII", "VIII", "IX"};
        printf("%s\n", roman_numerals[a - 1]);
    }
    
    return 0;
} 

File: Resources/TOI-Zero_68/A1/018/018.py
a = int(input())
if a > 9 or a == 0:
    print("Error : Out of range")
elif a < 0:
    print("Error : Please input positive number")
else:
    roman_numerals = ["I", "II", "III", "IV", "V", "VI", "VII", "VIII", "IX"]
    print(roman_numerals[a - 1])

    
